doc/source/reference/commands/status.rst

```diff
@@ -188,3 +191,11 @@ values:
              "version_patch": 0,
              "version": "2.0.0"
           }
+
+   * - ``n_workers``
+     - The value of ``n_workers`` set in this context.
+     - ``0``
+
+   * - ``default_n_workers``
+     - The default ``n_workers`` value of the Groonga process.
```

Comment:
TODO: Add a document for the common `n_workers` parameter to https://github.com/groonga/groonga/tree/main/doc/source/reference/command like https://github.com/groonga/groonga/blob/main/doc/source/reference/command/command_version.rst (but in English) and link from here. The document must describe the `GRN_N_WORKERS_DEFAULT` environment variable too.